FilecoinFoundationWeb/filecoin-foundation
# [UXIT-1716] Create Markdown Page Component

## 📝 Description
This PR introduces a refactor of how we handle markdown content in our application. It enables raw markdown content access, and introduces a reusable `MarkdownPage` component for consistent page layouts - ensuring that we can use our Markdown component mappings from the `MarkdownContent` component.

**Type:** Refactor

## 🛠️ Key Changes
- Enhanced Markdown loader configuration to support raw content access
- Included body export to TypeScript declarations for Markdown content
- Created reusable `MarkdownPage` component
- Updated existing Markdown pages to use the new `MarkdownPage` component

## 🧪 How to Test
### Test Cases
1. Markdown Content Access
   - Navigate to any markdown-based page
   - Verify content renders correctly
   - Check dev tools for proper structured data

2. Page Layout Consistency
   - Check Employee Privacy Policy page
   - Check Privacy Policy page
   - Check Terms of Service page
   - Verify consistent header and content layout

### Expected Results
- All Markdown pages should render with consistent layout
- Structured data should be present in page source
- No TypeScript errors related to Markdown types

## 📋 To-Do Before Merging
- [ ] Test all existing Markdown pages
- [ ] Verify structured data format
